astropy/coordinates/sky_coordinate.py

```diff
@@ -1486,22 +1486,55 @@ def radial_velocity_correction(self, kind='barycentric', obstime=None,
         care is needed in application. Strictly speaking, the barycentric correction is
         multiplicative and should be applied as::
 
-           sc = SkyCoord(1*u.deg, 2*u.deg)
-           vcorr = sc.rv_correction(kind='barycentric', obstime=t, location=loc)
-           rv = rv + vcorr + rv * vcorr / consts.c
+          >>> from astropy.time import Time
+          >>> from astropy.coordinates import SkyCoord, EarthLocation
+          >>> from astropy.constants import c
+          >>> t = Time(56370.5, format='mjd', scale='utc')
+          >>> loc = EarthLocation('149d33m00.5s','-30d18m46.385s',236.87*u.m)
+          >>> sc = SkyCoord(1*u.deg, 2*u.deg)
+          >>> vcorr = sc.radial_velocity_correction(kind='barycentric', obstime=t, location=loc)
```

Comment:
Any code affected by this line also need to be marked as `# doctest: +REMOTE_DATA`
```suggestion
          >>> vcorr = sc.radial_velocity_correction(kind='barycentric', obstime=t, location=loc)  # doctest: +REMOTE_DATA
```